app/api/db/db.py

```diff
@@ -0,0 +1,88 @@
+
+from datetime import datetime, timezone
+import os
+import uuid
+from aiohttp import ClientError
+from boto3.dynamodb.conditions import Key
+import boto3
+
+class DatabaseOperations:
+    def __init__(self):
+            self.dynamodb = boto3.resource('dynamodb', region_name=os.environ.get("AWS_REGION"))
+            self.football_results = self.dynamodb.Table('bs-football-results')
+            self.football_context_prompts = self.dynamodb.Table('bs-football-context-prompts')
+            self.user_contacts = self.dynamodb.Table('bs-user-contacts')
+
+    @classmethod
```

Comment:
Any reason to declare these methods as _static_ (@classmethod)?

Reply:
Static methods have better performance in the entire app. If you don't need an instance of a class, it's better to use static. Instantiating classes consumes memory, which have a lifetime. This way we can avoid that.
(the difference is small but it's better)